kernel/Base/MeshManipulatorBase.cpp

```diff
@@ -90,6 +90,12 @@ MeshManipulatorBase::MeshManipulatorBase(const ConfigurationData* config,
 
 MeshManipulatorBase::MeshManipulatorBase(const MeshManipulatorBase& other) =
     default;
+
+Zone* MeshManipulatorBase::addZone(std::string name) {
```

Comment:
Maybe we can return a reference, because it will not be `nullptr`

Reply:
Done